packages/workflow/src/interfaces.ts

```diff
@@ -165,19 +165,68 @@ export interface ContinueAsNewOptions {
   searchAttributes?: SearchAttributes;
 }
 
+/**
+ * Specifies:
+ * - whether cancellation requests are sent to the Child
+ * - whether and when a {@link CanceledFailure} is thrown from {@link executeChild} or
+ *   {@link ChildWorkflowHandle.result}
+ *
+ * @default {@link ChildWorkflowCancellationType.WAIT_CANCELLATION_COMPLETED}
+ */
 export enum ChildWorkflowCancellationType {
+  /**
+   * Don't send a cancellation request to the Child. Don't throw an error.
```

Comment:
Not sure this is right, I think that the error does get thrown but I'd need to verify.

I don't think we test all of these behaviors, I guess we will in the sdk-features repo.